fathom/src/surface/pretty.rs

```diff
@@ -576,19 +445,66 @@ impl<'interner, 'arena, A: 'arena> DocAllocator<'arena, A> for Context<'interner
                     RefDoc(&Doc::Nil),
                 )))
             }
+            _ => self.scope.to_scope(doc),
```

Comment:
Hmm, this seems to be breaking clippy?

Reply:
Was keeping it there temporarily to remind me to investigate it once I was done with everything else. It's been fixed now